tests/foreman/cli/test_discoveryrule.py

```diff
@@ -283,12 +276,17 @@ def test_negative_create_with_invalid_hostname(self):
         :CaseImportance: Medium
 
         :CaseLevel: Component
+
+        :BZ: 1378427
         """
         for name in self.invalid_hostnames_list():
             with self.subTest(name):
                 bug_id = name.pop('bugzilla', None)
```

Comment:
The nagger script will also look for BZs on collections like this `invalid_hostnames_list():` and notify if changes, so it addresses dynamic cases like this one